packages/core/test/w3c.test.ts

```diff
@@ -45,6 +45,16 @@ describe('5. Group', () => {
 });
 
 describe('7. Alias', () => {
+  test('top-level', () => {
```

Comment:
This was just paranoia, but I wanted to add a test that referencing an alias without any `.` in it worked. I just realized that’s untested. Fortunately that was unrelated to #191